src/cadenza.js

```diff
@@ -164,13 +166,20 @@ export class CadenzaClient {
    * @param {boolean} [options.hideWorkbookToolBar] - Whether to hide the workbook toolbar.
    * @param {string} [options.mediaType] - Set to "application/pdf" for Jasper Report views
    *     to show the PDF directly, without any Cadenza headers or footers.
+   * @param {FolderId} [options.highlightGlobalId] - The id of the folder to expand in the navigator.
```

Comment:
It's not just a Folder ID, it's a GlobalId. You can also highlight a non-folder tree item. Please change the type name and the description accordingly.